Input/Core/InputSystem.cs

```diff
@@ -15,7 +15,7 @@ static InputSystem()
 			s_Devices = new InputDeviceManager();
 
 			GameObject go = new GameObject("Input Prototype Controller");
-			go.hideFlags = HideFlags.HideAndDontSave;
+			go.hideFlags = HideFlags.DontSaveInEditor;//HideFlags.HideAndDontSave;
```

Comment:
Hmm..Wondering why I did this. It's possible it can be reverted.

Reply:
I believe this was a part of play mode. Maybe the object needs to be searched for now? I'll check to see what happens if it's reverted.